pkg/jsonutil/conversion.go

```diff
@@ -42,6 +43,11 @@ func JSONBytesToProto(contents []byte, m proto.Message) error {
 	return JSONUnmarshaler().Unmarshal(bytes.NewReader(contents), m)
```

Comment:
You can potentially change `JSONBytesToProto` and `JSONToProto` to function via newly created `JSONReaderToProto`:

```suggestion
	return JSONReaderToProto(bytes.NewReader(contents), m)
```